kivy/properties.pyx

```diff
@@ -1996,34 +2003,38 @@ cdef class ColorProperty(Property):
         Color value will be dispatched when set through indexing or slicing,
         but when setting with slice you must ensure that slice has 4 components
         with float values between 0-1.
+        Assingning color name as value is now supported.
+        Value `None` is allowed as default value for property.
     '''
 
-    def __init__(self, defaultvalue=None, **kw):
-        defaultvalue = defaultvalue or [1, 1, 1, 1]
+    def __init__(self, defaultvalue=0, **kw):
+        defaultvalue = \
+            [1.0, 1.0, 1.0, 1.0] if defaultvalue == 0 else defaultvalue
```

Comment:
Why support None? Why not just `if defaultvalue`?

Reply:
To skip empty collections like it is in `ListProperty` and to support `None` as default value (initial value) in case that user wants to use `allownone=True`.